sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/aio/__init__.py

```diff
@@ -109,6 +115,7 @@
     :type credential: ~azure.ai.textanalytics.TextAnalyticsApiKeyCredential or
         ~azure.core.credentials.TokenCredential
     :param str input_text: The single string to recognize entities from.
+        Limit text length to 5120 chars.
```

Comment:
Do we need to keep '5120' chars?

Reply:
Don't look too closely at the single text operation files since this PR will make them obsolete: #9639